build_tools/prepare_macos_wheel.sh

```diff
@@ -1,12 +1,11 @@
 #!/bin/bash
 
-set -e
-set -x
+set -exuo pipefail
 
-if [[ "$CIBW_BUILD" == *-macosx_arm64 ]]; then
+if [[ "${ARCHFLAGS:-}" == *arm64 ]]; then
     CONDA_CHANNEL="conda-forge/osx-arm64"
 else
     CONDA_CHANNEL="conda-forge/osx-64"
 fi
 
-conda create -n build -c $CONDA_CHANNEL jemalloc-local xsimd llvm-openmp
+conda create -y -n build -c $CONDA_CHANNEL jemalloc-local xsimd llvm-openmp
```

Comment:
```suggestion
mamba create -y -n build -c $CONDA_CHANNEL jemalloc-local xsimd llvm-openmp
```
?

Reply:
We don't have `mamba` installed in this workflow (yet). I can move to micromamba in a follow-up PR. It probably needs a bit of tweaking.